classes/AdminTab.php

```diff
@@ -864,6 +867,8 @@ public function displayListHeader($token = null)
      * @param string|null $token
      *
      * @deprecated 1.0.0
+     * @throws PrestaShopException
+     * @throws PrestaShopException
```

Comment:
Duplicate :-)

Reply:
Yeah, I noticed PhpStorm loves to duplicate it when my desktop hangs. Let's get that removed :)